src/shared-styles/shared-styles.html

```diff
@@ -17,10 +17,11 @@
       .debugging {
         background-color: #eeddee;
       }
-      @media only screen and (max-width: 700px) {
+      @media only screen and (max-width: 2400px) {
         img {
           display: block;
-          width: 80%;
+          width: 70%;
+          max-width: 600px;
```

Comment:
We definitely don't want to specify these properties on img in the shared-styles. This file is for styles shared across all views. What you're dealing with in this commit is one particular view, so put the style you want in that stylesheet.

Also, I doubt the media query is what we want either. Since we're designing for mobile screens, just use those conventions for now---no need for media queries. (Testing should always be done in the browser's device simulator so we keep the right layout in mind.)